time TimeStamp
string Iccid
float64 Latitude
string LatHemisphere
float64 Longitude
string LonHemisphere
float64 Altitude
float64 Accuracy
string UTCTime
string UTCDate
int32 Satellites
int32 Sim
int32 rssi
bool validGPS
string provider
string accessTechnology
string band
bool ModuleConnected
